tests/lib/rules/require-ember-lifeline.js

```diff
@@ -1,26 +1,31 @@
 const rule = require('../../../lib/rules/require-ember-lifeline');
-const MESSAGE = rule.meta.message;
+const getMessage = rule.meta.message;
 const RuleTester = require('eslint').RuleTester;
 const ruleTester = new RuleTester();
```

Comment:
I'm happy to refactor this, just want it to be in a separate PR. I created #65 to address this.